manifests/webhook.yaml

```diff
@@ -116,9 +162,9 @@ webhooks:
     failurePolicy: Fail
     clientConfig:
       service:
-        name: admission-controller-svc
-        namespace: ${NAMESPACE}
-        path: "/statefulsets"
+        name: kubernetes
+        namespace: default
+        path: "/apis/admission.tidb.pingcap.com/v1alpha1/statefulsetadmissionreviews"
```

Comment:
use a single path for all webhooks: `/apis/admission.tidb.pingcap.com/v1alpha1/admissionreviews`.

Reply:
issued in https://github.com/pingcap/tidb-operator/issues/1316